README.md

```diff
@@ -25,6 +25,33 @@ git clone https://github.com/usdot-fhwa-stol/v2x-ros-conversion.git
 cd v2x-ros-conversion/docker
 ./build-image.sh
 ```
+
+To incorporate the v2x-ros-conversion package into a docker project, call the following command to clone the v2x-ros-conversion package into the project:
+```
+git clone --depth=1 https://github.com/usdot-fhwa-stol/v2x-ros-conversion.git ${dir}/src/v2x-ros-conversion --branch "develop-foxy"
+```
+Once installed, the required dependencies for the v2x-ros-conversion package need to be installed prior to building the package. To do so, run 
+```
+sudo ~/src/v2x-ros-conversion/docker/install_dependencies.sh
+```
+
+Once the dependencies are installed, build the image as shown above.
+
+### Build From Source
+Ensure that a version of Ubuntu 22 is installed with the foxy version of carma-base. Once installed, follow the following steps in install the v2x-ros-conversion package into the project.
```

Comment:
We need ubuntu 20.04 (Focal Fossa) for foxy. Also when we are building from source we dont need carma-base at all. That is a base image for docker builds. 
This brings up a good point - will build from source work without carma-base installed packages? Can you test this out, if it doesn't we want to only include suggested docker based deployment here